src/HL-phpclient/HLMembers.php

```diff
@@ -129,4 +129,17 @@ public function import($listId,$params,$file)
         $this->endpoint = 'lists/'.$listId.'/import';
         return $this->call('POST',$this->endpoint,$params,$file);
     }
+
+    /**
+     * Obscure member on list.
+     * @param $listId
+     * @param $memberId
+     * @return mixed
+     */
+    public function obscure($listId, $memberId)
```

Comment:
Why not make a more generic approach instead of adding methods for every function provided on the API?

Reply:
LR-4510 is created